Doc/tools/templates/layout.html

```diff
@@ -1,4 +1,14 @@
 {% extends "!layout.html" %}
+{% block header %}
+{%- if outdated %}
+<div id="outdated-warning" style="padding: .5em; text-align: center; background-color: #FFBABA; color: #6A0E0E;">
+    {% trans %}This document is for an old version of Python that is {% endtrans %}<a href="https://devguide.python.org/devcycle/#end-of-life-branches">{% trans %}no longer supported{% endtrans %}</a>.
+    {% trans %}You should upgrade, and read the {% endtrans %}
+    <a href="/3/{{ pagename }}{{ file_suffix }}">{% trans %} Python documentation for the current stable release{% endtrans %}</a>.
```

Comment:
Could you try adding `{{ language or 'en' }}` like below on line 8/18?

Reply:
I tried this out and it almost works, but https://docs.python.org/en/3/ isn't a working URL the way https://docs.python.org/ja/3/ and https://docs.python.org/pt-br/3/ are.

I used the ternary operator to get something which is unelegant but works. I tested it three ways: 
* with no language set: `make html 'SPHINXOPTS=-A outdated=1'`
* with the language set to 'en': `make html 'SPHINXOPTS=-A outdated=1 -A language=en'`
* with the language set to a value other than 'en': `make html 'SPHINXOPTS=-A outdated=1 -A language=pt-br'`

Let me know what you think.

Reply:
Sounds great to me, but I’m not the expert on Python docs translations, I’ll tag Julien.